internal/middleware/requestid_test.go

```diff
@@ -0,0 +1,64 @@
+package middleware
+
+import (
+	"context"
+	"testing"
+
+	grpc_testing "github.com/grpc-ecosystem/go-grpc-middleware/testing"
+	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/testing/testproto"
+	"github.com/openfga/openfga/pkg/logger"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"google.golang.org/grpc"
+)
+
+var pingReq = &pb_testproto.PingRequest{Value: "ping"}
+
+type pingService struct {
+	pb_testproto.TestServiceServer
+	T *testing.T
+}
+
+func (s *pingService) Ping(ctx context.Context, req *pb_testproto.PingRequest) (*pb_testproto.PingResponse, error) {
+	_, ok := ctx.Value(requestIDCtxKey).(string)
```

Comment:
We could get double coverage here if you use the context helper methods instead.

```
_, ok := RequestIDFromContext(ctx)
require.True(s.T, ok)
```

Reply:
Done! Thanks!